### PR TITLE
Fix timeline for one host [fixes #12]

### LINKED ISSUE #12: Fix timeline for one host
screen shot 2016-11-19 at 12 55 05

### PR DESCRIPTION
**Pull request by sseliverstov**

### CODE REVIEW

**Comment by baev** [comment]
@just-boris could you please take a look

File: face/src/plugins/timeline/TimelineView.js [hunk 1]
```diff
@@ -43,59 +47,90 @@ class TimelineView extends BaseChartView {
...
+        this.x.domain([0, this.maxX]).nice();
         this.x.range([0, width]);
 
-        this.miniX.domain([0, to - from]).nice();
+        this.miniX.domain([0, this.maxX]).nice();
         this.miniX.range([0, width]);
 
         this.svg = this.setupViewport();
 
+        var total_threads = this.model.get('children').reduce((a,b) => { return a + b.children.length; }, 0);
```
Comment (just-boris) [inline face/src/plugins/timeline/TimelineView.js]: minor issue, but we are using `camelCase` identifiers

**Comment by sseliverstov** [comment]
@just-boris change name with camelCase

**Comment by just-boris** [comment]
Actually, the code was changed so much since I touched that last time, so I just checked that it looks like it is supposed to be, not deeply looking into details.

**Comment by baev** [comment]
@sseliverstov thanks!

**Pull request merged**